### DISCUSSION TITLE
Duckstation for Mac M1 presents some issues for some games.

### DISCUSSION THREAD

**Discussion by ThinAaronGB** [community/community#37979] [Other Feature Feedback, Questions, & Ideas]
Hi! First, I'd like to say that your emulator is so cool! I've been using it since May of 2022 and I can't even tell you how impressive this emu is! The quality of the games in 4k plus the Vulkan system you operate in your emulator is out of this world! Thank you for all this really. I stopped using Duckstation for some time though, and I ended up deleting it altogether, then around September, I decided I wanted to reinstall it for the second time so off I went, however, this time some titles that I love one of them being Silent Hill 1 began to crash while playing. It automatically took me out of the game with no apparent reason something that had never happened back then in May, (first time I tried Duckstation on my MacBook Air M1) plus some other issues like in some video clips suddenly everything freezes out. I honestly would like to continue to use this emulator as in my opinion it's the best PS1 emulator under the sun and I love the design and the aesthetics of the games thanks to Duckstation and thanks to all of you who have made this possible. I hope you can work this/these issue(s) out. I don't know why this happens but I am 100 percent sure it did not happen before. Especially with this one game Silent Hill among some other titles too. Thank you for your time! Please let me know how I can contribute to this project so that I can help you somehow. 
Kindest regards! Aaron.

**Comment by ossanna16** [comment]
Hi @ThinAaronGB,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Duckstation. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
